lib/node_modules/@stdlib/string/base/replace-before/docs/repl.txt

```diff
@@ -0,0 +1,31 @@
+
+{{alias}}( str, search )
+    Replaces the substring before the first occurrence of a specified search string.
+
+    Parameters
+    ----------
+    str: string
+        Input string.
+
+    search: string
+        Search string.
+    
+    replacement: string
+        Replacement string.
+
+    Returns
+    -------
+    out: string
+        string.
+
+    Examples
+    --------
+    > var str = 'beep boop';
+    > var replacement = 'foo';
+    > var out = {{alias}}( str, ' ', replacement )
+    'fooboop'
+    > out = {{alias}}( str, 'o', replacement )
+    'foooop'
+
+    See Also
+    --------
```

Comment:
IIRC, we should have another trailing newline here (i.e., two trailing newlines).